xbrowse_server/api/views.py

```diff
@@ -612,11 +613,9 @@ def add_or_edit_variant_tags(request):
             form.cleaned_data['xpos'],
             form.cleaned_data['ref'],
             form.cleaned_data['alt'],
+            force_create=True
```

Comment:
I think it's better to leave the `force_create` logic outside of `get_single_variant(..)` since it only occurs in 2 places. 
A `get` method with 2 possible return values (the found variant or None) seems easier to understand.